templates/web/pages/product/forest_footprint_details.tt.html

```diff
@@ -0,0 +1,35 @@
+<p>
```

Comment:
```suggestion
<!-- [% template.name %] -->
<p id="forest_footprint_calculation_details">
```

[% template.name %] should print the template name.

Reply:
Perfect, I didn't know [% template.name %]